Packs/VMRay/ReleaseNotes/1_1_5.md

```diff
@@ -0,0 +1,2 @@
+##### VMRay
+*Note: Support for this pack moved to the Partner on December, 21, 2021. Please contact the Partner directly via the support link on the right.*
```

Comment:
Make this a bit clearer and shorter, full line can show in the readme as @kgal-pan suggested :)
```suggestion
- Support moved to partner on December 21, 2021
```